### PR TITLE
stringGlues

### PR DESCRIPTION
**Pull request by beccers2017**
Added a function stringGlues

### CODE REVIEW

File: math/stringGlues.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	
+	return full + v.pop(); 
+}
+
+int main() {
+	vector<string> v = {"dog", "cat", "house"};
+	cout << stringGlues(v);
+	
+	
+}
```
Comment (Soat6041) [inline math/stringGlues.cpp]: all i see missing is a semi colon on your main otherwise great work

File: math/stringGlues.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <string>
+#include <vector>
+using namspace std;
+
+string stringGlues(vector<string> v) {
```
Comment (muzny) [inline math/stringGlues.cpp]: Follow our style guidelines: this should be `StringGlues`.

Add a function comment explaining what this function does, what its parameters are, and what its return value is.

File: math/stringGlues.cpp [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	string full=""; 
+	
+	for(int i=0; i < v.length() - 1; i++)
+	{
+		full = full+v[i];
+		full= full+"glue";
```
Comment (muzny) [inline math/stringGlues.cpp]: inconsistent spacing here

File: math/stringGlues.cpp [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	}
+	
+	return full + v.pop(); 
+}
+
+int main() {
```
Comment (muzny) [inline math/stringGlues.cpp]: Move your StringGlues function into FunMath.h (declaration) and FunMath.cpp (implementation) and remove the file stringGlues.cpp.

**Pull request merged**